### PR TITLE
Travis CI

### PR DESCRIPTION
**Pull request by YannickFricke**
Added support for Travis CI and added gradle wrapper

### CODE REVIEW

**Comment by Sanduhr32** [comment]
Well i dont recommend using Gradle wrapper 4.3.1 yet because i had some issues with it already.

**Comment by YannickFricke** [comment]
No problem at my local machine and on travis ci

**Comment by YannickFricke** [comment]
@Sanduhr32 did you tested it locally?
When it "shouldnt work", why does it works for me and on travis ci?

What is your error?

**Comment by Sanduhr32** [comment]
My current build failed again. I'm sorry. I'm sure that it will be fixed later but if the build fails we cant run it.
My error is an IllegalStateException if i try to build the shadowJar :/

**Comment by YannickFricke** [comment]
Maybe a problem with your local environment?

**Comment by Sanduhr32** [comment]
I'm trying to annoy duncte hard enough so he tests it. I'll approve if he has no issues because i heard some contacts talking about the same error. Regards Sanduhr.

**Comment by YannickFricke** [comment]
@Sanduhr32 @duncte123 What do you think about maven?
I could rewrite the build script to use maven

**Comment by Sanduhr32** [comment]
I dont think that we would use Maven because we never talked about using it.
Duncte said he will check it next morning because its 11:30pm.

**Comment by YannickFricke** [comment]
Just because you have not talked about it, that's no reason not to use it...

**Comment by Sanduhr32** [comment]
You are right @YannickFricke but sadly i have no good experiences with Maven. Only trouble and with gradle its not the annoying dependency management issue. And dont ask me about SBT i never really used it because i dont like scala that much but thats more a personal thing because some scala devs poked me for using normal Java or using Kotlin until i started ignoring them. I mean we could give it a try but thats not my decision. Your PR looks very good and professional. I pay you my daily respect.

**Comment by duncte123** [comment]
@YannickFricke I'm not using maven because it's nothing more than a big compiler issue, nevet had anything but problems with it

**Comment by Sanduhr32** [comment]
I cleared my Gradle Cache now its working. I'm sorry for interrupting.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,58 @@ The following libraries are used:
...
+# How to setup?
+## Replace the X.X.X with the latest version
+Download the latest release here: [Releases](https://github.com/YannickFricke/SkyBot/releases)
+You need the `skybot-X.X.X-all.jar`
+
+Execute the following command
+```shell
+java -jar skybot-X.X.X-all.jar
+```
+The bot will create a file named `config-template.json`.
```
Comment (duncte123) [inline README.md]: The file is called `config.json` not `config-template.json`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# SkyBot
+# SkyBot [![Build Status](https://travis-ci.org/YannickFricke/SkyBot.svg?branch=travis)](https://travis-ci.org/YannickFricke/SkyBot)
```
Comment (ramidzkh) [inline README.md]: This needs to be fixed later if accepted

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+language: java
+dist: precise
+jdk:
+- oraclejdk8
+script: sh gradlew build
+deploy:
+  provider: releases
+  skip_cleanup: true
+  api_key:
+    secure: a0MFnxh0c9O8hV6LEK+iDDPVuFRUUXwhrpZaPq6qcD+kq/CTjCz+wCUhR7cm6FK9lI0VShxDaQjVsrSOlTL/fEZKszhGwStWV5uG10AOG/B4A6VX1NsCGCocvt3RKZtoZN4uyWBL7LjY9Vca6Hl1x6/7y+wOsg/4HPghmJk3Bhda8pPUHEzOYXoTITgY220jbP6wNLyiex/GKYHiFY6XBfMIfw6K9yWriQ/DUDlRX40POM/6TC4bXhxaKjjFC/x5HiFUFRzpvpE34utLL6wGTUVruJblTk14YMlwlRzp4ReQYzCHUN/LAOWLjXp/nTbtnXBkutV8cA1a4QJWgEiGlgRrrmmdlKIiK7KfuvqOcy3LmRSASkAjTO5uY0vc6DbqV1caXslGYRawSWmg4bOFoVqWAY89+gWHLFCFI30OIPX5uleyXwdS0x4m3jA8dPT7Im4CSJ5EYoicqOrYh3hCTjyxL6mSyqdfSjFIhtfLwT42Cbb/OnXX1VrN9f0zT56FlXFtAqJKhHKcSYQOWYqpLyp17F7J4S2fHZPtj9Ax1roCYiTRoBXI5QgBp2yz+ELqgrPwTsuBi5O8hGcMcpFfLFOQNMXkO/RDM5tqIxKNSZkXimky/f/xwSWpNSsr7ASG80fKC/RcT0gZActJjucIU01IFz+eS2QuKNPbXTWX9f8=
```
Comment (ramidzkh) [inline .travis.yml]: What does this do?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  file:
+    - skybot-3.50.3.jar
```
Comment (ramidzkh) [inline .travis.yml]: This file doesn't have any dependencies so it wouldn't work

File: README.md [hunk 5]
```diff
@@ -7,6 +7,58 @@ The following libraries are used:
...
+Needed entries:
+
+```json
+{
+  "discord": {
+    "token": "<insert your discord token here>",
+    "prefix": "/",
+    "totalShards": 1
+  },
+  "use_database": false,
```
Comment (ramidzkh) [inline README.md]: Elaborate on the MySQL and SQLite function

File: README.md [hunk 6]
```diff
@@ -7,6 +7,58 @@ The following libraries are used:
...
+    "wolframalpha": ""
+  }
+}
+```
+
+Go [here](https://discordapp.com/developers/applications/me) and add a new application. After that click on `Create a Bot User` and copy the token and replace it in `config-template.json`.
+
+In the above example you would replace `<insert your discord token here>` with the generated token by Discord.
+
+After that rename `config-template.json` to `config.json`.
```
Comment (ramidzkh) [inline README.md]: The file is automatically generated. You can CTRL C to close it then edit the file yourself

File: README.md [hunk 7]
```diff
@@ -7,6 +7,54 @@ The following libraries are used:
...
+Edit this file with your settings
+
+Needed entries:
+
+```json
+{
+  "discord": {
+    "token": "<insert your discord token here>",
+    "prefix": "/",
+    "totalShards": 1
```
[the remark this answers is not in the document]
Comment (YannickFricke) [reply README.md]: It's the default of the bot...

File: .travis.yml (hunk 4 above)
Comment (YannickFricke) [reply .travis.yml]: It would work, when the user has all dependencies in his classpath

File: README.md (hunk 5 above)
Comment (YannickFricke) [reply README.md]: IDK how its integrated in the bot, so someone of you should document it

Comment (Sanduhr32) [reply README.md]: Yeah _we_ (**ramidzkh**) will document it!

Comment (duncte123) [reply README.md]: @ramidzkh when will you write that wiki about the SQL?

**Pull request merged**